content-src/selectors/selectors.js

```diff
@@ -59,20 +59,19 @@ const selectSpotlight = module.exports.selectSpotlight = createSelector(
         return -1;
       }
     });
-    return Object.assign({}, FrecentHistory, {rows});
+    return Object.assign({}, Highlights, {rows});
   }
 );
 
 module.exports.selectNewTabSites = createSelector(
   [
     state => state.TopSites,
-    state => state.FrecentHistory,
-    state => state.History,
     state => state.Highlights,
```

Comment:
Unless we want to make the debugger use `Spotlight`, which will filter blocked links and stuff